packages/edit-site/src/components/global-styles/font-library-modal/font-card.js

```diff
@@ -28,8 +28,7 @@ function FontCard( { font, onClick, variantsText, navigatorPath } ) {
 
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> typogrpahy --> open font library modal.

**Note:** This has changed some styles, but that is not looking good, should we add some higher specificity or !important? as we did previously with block library? @mirka @ciampo 

| Before | After     |
| :-------- | :------- |
| <img width="1327" alt="Font-library-modal-before" src="https://github.com/user-attachments/assets/282fbd04-6ee1-4854-954e-c5522d0fd544"> | <img width="1327" alt="font-librayr-modal-after" src="https://github.com/user-attachments/assets/cf3e129a-9d24-4583-86b2-8a3d7b5098fa"> |

Reply:
Yes, I guess we can go with a `!important` on the `height: auto`?

Reply:
(FYI, unrelated to this but the tabs styling was broken. Fix proposed in #65330)

Reply:
> Yes, I guess we can go with a `!important` on the `height: auto`?

Sure, Would do this.